.github/workflows/major-release.yml

```diff
@@ -0,0 +1,49 @@
+name: Major Release
+
+permissions:
+  issues: write
+  contents: write
```

Comment:
We might want to move this to the job level and keep "global" permission to the stricter `content: read` as was done in e.g. https://github.com/nodejs/node/blob/7f392d4a144ac4b2f4a42afffd05bf19c479fe31/.github/workflows/auto-start-ci.yml#L16-L22

Also, do we need `contents: write`?